ts/src/pro/poloniex.ts

```diff
@@ -622,67 +622,10 @@ export default class poloniex extends poloniexRest {
         for (let i = 0; i < data.length; i++) {
             const order = this.safeValue (data, i);
             const marketId = this.safeString (order, 'symbol');
-            const eventType = this.safeString (order, 'eventType');
             if (marketId !== undefined) {
```

Comment:
Can you be a bit more specific pls? What exactly isn't working here?

Reply:
This approach only makes sense when there is specific trade information given  alongside the order response, like in [hitbtc](https://github.com/ccxt/ccxt/pull/18027) (I'll make a unified method for exchanges like this), these exchanges give order updates, we're not actually getting the price for each trade, because we're only given the overall price for the order so it doesn't make sense to calculate an average price/cost. We also aren't calculating the amount of the trades correctly. These responses given the total amount filled, so the amount of each trade should be the total fill - the total fill amount from the last update.

There's [another PR](https://github.com/ccxt/ccxt/pull/18082) that I'm working on which calculates some trade information based on order updates, but I expect it to take a while to get merged and that we should remove this bad logic before that happens